text/0000-customisability.md

```diff
@@ -0,0 +1,80 @@
+- Start Date: 2016-10-25
+- RFC Issue: #3
+- RFC PR: 
+- Implementation Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+Customisation of Rustfmt should be allowed (via a `rustfmt.toml` file), but not
+encouraged.
+
+
+# Details
+[details]: #details
+
+Users may create a `rustfmt.toml` or `.rustfmt.toml` in their project directory
```

Comment:
I think we should break it, or at least not use it for this config. It looks odd. Rubocop's default config is named `.rubocop.yml`, and CoffeeLint's deafult config is named `coffeelint.json`. Generally, lowercase is much nicer since it's easier to type IMHO. (The argument for Cargo.toml being sorted similar to Makefile makes sense though.)